src/app.scss

```diff
@@ -189,6 +189,12 @@
     margin-inline-end: 0;
 }
 
+// Fix the double-spacing issue for other dropdowns
+// https://github.com/patternfly/patternfly/issues/6632
+.pf-v5-c-menu-toggle__controls:last-child > .pf-v5-c-menu-toggle__toggle-icon {
```

Comment:
This will make regular `MenuToggle`s look "wrong", see https://github.com/cockpit-project/cockpit/pull/21356#issuecomment-2516538987. I think it would be better to make this apply only to the MenuToggles where you explicitly want this, like the bookmarks dropdown.